docs/Endpoints.md

```diff
@@ -123,14 +123,13 @@ Instead of modifying the request, you could simply log it, instead.
 
 ```swift
 let requestClosure = { (endpoint: Endpoint<GitHub>, done: MoyaProvider.RequestResultClosure) in
-    do {
-        var request = try endpoint.urlRequest()
-        // Modify the request however you like.
-        done(.success(request))
-    } catch {
-	done(.failure(MoyaError.underlying(error)))
```

Comment:
@devSC Thanks for this pull request, however, this is the correct code style for Moya `10.0`+.

Reply:
oh, i mistake.